### DISCUSSION TITLE
Welcome to termux Discussions!

### DISCUSSION THREAD

**Discussion by joseguzman1337** [4k4xs4pH1r3/termux#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eldamsheety** [comment]
Hi ... I'm Omar from Egypt ... I installed Termux on my phone and I also installed some packages on it ... could you please give me a list of all packages which I should install.
